docs/DEVELOPING.md

```diff
@@ -82,11 +82,13 @@ The connector uses the Elastic ingest attachment processor plugin for extracting
 
 ### E2E Tests
 
-The framework allows users to test the connector end to end. To perform e2e test for Network Drive connector, run the following make command:
+The end to end test is usually performed by developers after the functional and system testing is completed. The framework allows users to test the connector end to end. To perform e2e test for Network Drive connector, run the following make command:
 ```shell
 $ make ftest NAME=network_drive
 ```
 
+ℹ️ Users do not need to have a running Elasticsearch instance or a Network Drive source to run this test. The docker compose file manages the complete setup of the development environment, i.e. both the mock Elastic instance and mock Network Drive source using the docker image.
```

Comment:
For this statement in both network drives and S3 section, I think it will be helpful to call out that the e2e test does not currently support users running against an Elastic Cloud instance (or an Elasticsearch instance that is not the mocked Elasticsearch instance already created by the docker compose file) until https://github.com/elastic/connectors-python/issues/356 is resolved.

Reply:
updated with commit:
 [resolve comments](https://github.com/elastic/connectors-python/pull/385/commits/f05b2b2b7fda74556d664f89d2092a072dc21a85)